### ISSUE TITLE
interested in making this a wordpress compatible cms?

### ISSUE THREAD

**Issue by superdolt** [vladimirok5959/golang-fave#8]
so people can just export their db from wordpress and go inside golang-fave

are u still developing this? need help?

**Comment by vladimirok5959** [comment]
Yes, still in development. This will be not easy. Currently CMS don't supports blog post comments. How to be with attachments (blog post files, images)? If you want, you can try to make standalone server side script for that or start making new module inside project with admin section, but I think CMS is not fully ready for that. Thanks for interesting for my project :)

**Comment by superdolt** [comment]
1. how about we do one by one translation of wordpress php to golang?
2. we can forgo the comment section for now.
3. are you interested in this? u can lead or i can coz at this stage, if it's not wordpress compatible (for exporting db to this), it's not really worth taking forward all the way. we can make it into an alternative wordpress project.

**Comment by vladimirok5959** [comment]
> 1. how about we do one by one translation of wordpress php to golang?
That's is bad idea, you can simply run Wordpress PHP-FPM with Nginx and website will be works almost with the same speed

> 2. we can forgo the comment section for now.
Ok

> 3. are you interested in this?
No, sorry, I have little different issues to finish with Shop module, blog posts images and others, when I will have time. But it's MIT and open source, you can fork project and create pull request
